### PR TITLE
docs: add NGINX Plus

### PR DESCRIPTION
**Pull request by SkipperQ**

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Selflocking

File: docs/integration/C++/NGINX Plus.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
+title: NGINX Plus
+description:  Using Casdoor in NGINX Plus
+keywords: [NGINX Plus]
+authors: [SkipperQ]
+---
+
+# NGINX Plus
```
Comment (Selflocking) [inline docs/integration/C++/NGINX Plus.md:8]: image

File: docs/integration/C++/NGINX Plus.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    ![subUsers](/img/integration/C++/NGINX_Plus/subUsers.png)
+    
+
+## Configuring NGINX Plus
+
+Configure NGINX Plus as the OpenID Connect relying party:
+
+1. Create a clone of the **[nginx-openid-connect](https://github.com/nginxinc/nginx-openid-connect)** GitHub repository.
+    
+    ```
```
Comment (Selflocking) [inline docs/integration/C++/NGINX Plus.md:101]:
```suggestion
    ```bash
```

File: docs/integration/C++/NGINX Plus.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    
+2. Copy these files from the clone to **/etc/nginx/conf.d**:
+    - **frontend.conf**
+    - **openid_connect.js**
+    - **openid_connect.server_conf**
+    - **openid_connect_configuration.conf**
+3. Get the URLs for the authorization endpoint, token endpoint, and JSON Web Key (JWK) file from the Casdoor configuration. Run the following `curl` command in a terminal, piping the output to the indicated `python` command to output the entire configuration in an easily readable format. We’ve abridged the output to show only the relevant fields. <span id="jump1"></span>
+
+    
+    ```
```
Comment (Selflocking) [inline docs/integration/C++/NGINX Plus.md:113]:
```suggestion
    ```bash
```

**Comment by Selflocking** [comment]
image

There should be no spaces in the file name. Please rename to `NGINX-Plus.md`

**Review by Selflocking — CHANGES_REQUESTED** [review]

**Comment by Selflocking** [comment]
@SkipperQ You can megre this PR(https://github.com/SkipperQ/casdoor-website/pull/2) to pass these checks.
image

**Comment by Selflocking** [comment]
https://github.com/SkipperQ/casdoor-website/pull/3

**Comment by Selflocking** [comment]
@SkipperQ please sign our [Contributor License Agreement](https://cla-assistant.io/casdoor/casdoor-website?pullRequest=523)

image

**Pull request merged**